### PR TITLE
Jealousy/Dislike grief

### PR DESCRIPTION
**Pull request by Orakoro**
Added 1-2 new grief scenarios for child jealousy and dislike and general jealousy and dislike

### CODE REVIEW

**Review by Lixxis — APPROVED** [review]
I'm not quite familiar with the Pronouns Tagging, so I would love to get another eye on the PR.

File: child_dislike.json [hunk 1]
```diff
@@ -0,0 +1,376 @@
+{
+  "general": {
+    "body": [
+      "It's sad, the vigil. More so because r_c knows {PRONOUN/r_c/subject} should care and doesn't, than any grief for m_c.",
+      "r_c refuses to sit vigil for m_c. {PRONOUN/m_c/subject/CAP} failed r_c as a parent, guardian, and mentor. Why should r_c honor {PRONOUN/m_c/object} now?",
+      "r_c is called to m_c's vigil, but declines to speak. {PRONOUN/r_c/subject/CAP}{VERB/r_c/'re/'s} not sure {PRONOUN/r_c/subject}'d have anything good to say.",
+      "They may have shared blood, but nothing more. r_c doesn't show up to m_c's vigil."
+,      "Even though m_c wasn't the best parent out there, r_c knew that {PRONOUN/r_c/subject} had the decency to be respectful at m_c's vigil."   ],
```
Comment (scribblecrumb) [inline child_dislike.json]: feels a bit like r_c is saying they know *m_c* would have the decency to be respectful.  It might be good to reword this as "m_c wasn't the best parent out there, but r_c decided that {PRONOUN/r_c/subject} had the decency to at least be respectful at m_c's vigil."

File: child_jealousy.json [hunk 2]
```diff
@@ -0,0 +1,376 @@
+{
+  "general": {
+    "body": [
+      "It's sad, the vigil. More so because r_c knows {PRONOUN/r_c/subject} should care and doesn't, than any grief for m_c.",
+      "r_c refuses to sit vigil for m_c. {PRONOUN/m_c/subject/CAP} failed r_c as a parent, guardian, and mentor. Why should r_c honor {PRONOUN/m_c/object} now?",
+      "r_c is called to m_c's vigil, but declines to speak. {PRONOUN/r_c/subject/CAP}{VERB/r_c/'re/'s} not sure {PRONOUN/r_c/subject}'d have anything good to say.",
+      "They may have shared blood, but nothing more. r_c doesn't show up to m_c's vigil.",      "Despite what {PRONOUN/r_c/poss} family would say, m_c was not the wonderful parent that r_c wanted {PRONOUN/m_c/subject} to be, and {PRONOUN/r_c/subject} don't shed a single tear at m_c's vigil."
```
Comment (scribblecrumb) [inline child_jealousy.json]: second pronoun should be object, not subject

File: general_dislike.json [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+  "general": {
+    "body": [
+      "Nobody wants to admit such a terrible thing out loud, but m_c will not be missed by the whole Clan. r_c wanders past the vigil being held for {PRONOUN/m_c/object}, pointedly ignoring it.",
+      "r_c approaches the clearing where m_c's body lies, cold and lifeless. Defying expectations of sympathy and grief, r_c instead curses {PRONOUN/r_c/poss} fallen Clanmate to the Dark Forest where {PRONOUN/m_c/subject} {VERB/m_c/belong/belongs}.",
+      "r_c wanders past m_c's vigil. {PRONOUN/r_c/subject/CAP} won't mourn m_c's passing.",
+      "Frankly, they're all better off without m_c, and r_c doesn't know why everyone else is so grief stricken.",
+      "As much as {PRONOUN/r_c/subject} hated m_c, the sight of {PRONOUN/m_c/poss} body still fills r_c with dread.",
+      "Yes, {PRONOUN/r_c/subject} hated m_c, but m_c was {PRONOUN/r_c/poss} Clanmate first and foremost, and r_c bows {PRONOUN/r_c/poss} head at the vigil respectfully.",      "Despite feeling uncomfortable at m_c's vigil, r_c understands how everybody would feel if {PRONOUN/r_c/subject} bad-mouthed {PRONOUN/r_c/poss} own Clanmate, therefore r_c kept quiet in the wishes of m_c's family."
```
Comment (scribblecrumb) [inline general_dislike.json]: this is worded a little confusingly and is a little too wordy, might be good to reword as: "Despite feeling uncomfortable at m_c's vigil, r_c knows how everyone would react if {PRONOUN/r_c/subject} bad-mouthed {PRONOUN/r_c/poss} own Clanmate, so {PRONOUN/r_c/subject} {VERB/r_c/keep/keeps} quiet."

File: general_jealousy.json [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+      "r_c watches the vigil being held for m_c from the sidelines, not at all sure how {PRONOUN/r_c/subject} {VERB/r_c/feel/feels} about {PRONOUN/m_c/poss} death.",
+      "r_c resented m_c for what {PRONOUN/r_c/subject} couldn't have, but seeing m_c's body lying in the clearing as the Clan hold vigil for {PRONOUN/m_c/object} makes {PRONOUN/r_c/poss} jealousy feel petty and stupid.",
+      "r_c has always felt jealous of m_c, but could never want to take {PRONOUN/m_c/poss} place now, watching the Clan holding vigil for {PRONOUN/m_c/object}.",
+      "r_c listens to the stories being passed around as the vigil for m_c continues, and stalks off to {PRONOUN/r_c/poss} nest. {PRONOUN/r_c/subject/CAP} can't stand hearing everyone praise such a useless cat.",
+      "m_c really wasn't that great, but it's amazing how many cats want to ignore that now that {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} dead, and r_c isn't interested in hearing it."
+
+,      "r_c knows that even though m_c's family and friends are present at the vigil, everybody has a right to speak up, and so {PRONOUN/r_c/subject} {VERB/r_c/tell/tells} everyone at the vigil just what {PRONOUN/r_c/subject} think of m_c."    ],
```
Comment (scribblecrumb) [inline general_jealousy.json]: needs a comma after "r_c knows that"

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
just a few rewording suggestions and grammar fixes

**Comment by Orakoro** [comment]
> just a few rewording suggestions and grammar fixes

i've made the changes you want, do tend to word things quite strangely so i'll make sure to read over contributions in future when i make more ^^

**Review by scribblecrumb — APPROVED** [review]
